numba/cuda/cudadrv/nvvm.py

```diff
@@ -674,7 +674,11 @@ def _replace_llvm_memset_usage(m):
     Used as functor for `re.sub.
     """
     params = list(m.group(1).split(','))
-    align = re.search(r'align (\d+)', params[0]).group(1)
+    align_attr = re.search(r'align (\d+)', params[0])
+    if not align_attr:
+        raise ValueError("No alignment attribute on memset dest")
```

Comment:
```suggestion
        raise ValueError("No alignment attribute found on memset dest")
```
I think `found` is needed to hint that one is required?